protoc-gen-gogo/generator/generator.go

```diff
@@ -3280,15 +3280,7 @@ func (g *Generator) generateInitFunction() {
 }
 
 func (g *Generator) generateFileDescriptor(file *FileDescriptor) {
-	// Make a copy and trim source_code_info data.
-	// TODO: Trim this more when we know exactly what we need.
-	pb := proto.Clone(file.FileDescriptorProto).(*descriptor.FileDescriptorProto)
-	pb.SourceCodeInfo = nil
-
-	b, err := proto.Marshal(pb)
-	if err != nil {
-		g.Fail(err.Error())
-	}
+	b := g.Buffer.Bytes()
```

Comment:
Does this look correct?

The issue I run into is that protoc-gen-gogo modifies the input .proto files (see [example 1](https://github.com/cosmos/gogoproto/blob/eb0de473e08157c23d511839c4eebb1fefd364b3/protoc-gen-gogo/descriptor/descriptor_gostring.gen.go#L369), [example 2](https://github.com/cosmos/gogoproto/blob/eb0de473e08157c23d511839c4eebb1fefd364b3/protoc-gen-gogotypes/main.go#L43)), then marshals the modified object. This is super annoying, because on our own SDK proto files, we have mismatches. I'm proposing that the file_descriptor_bytes are the gzipped bytes of the raw input .proto file.